### PR TITLE
Prevent request warnings for tests of 4xx and 5xx responses

### PR DESCRIPTION
**Pull request by jsugarman**
Prevent log warnings for tests of 4xx and 5xx responses

### CODE REVIEW

**Comment by jsugarman** [comment]
> I could be really strict about PEP8 code style on this if you want, but it's only super trivial stuff, basically 👍

Would be good for me to learn and this PR was trivial in any event. Is it indentation related? feel free to comment against the code if you have time

File: fee_calculator/apps/api/tests/test_advocate_type_api.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 from calculator.constants import SCHEME_TYPE
 from calculator.models import Scheme
-
+from calculator.tests.lib.utils import prevent_request_warnings
```
Comment (ibrechin) [inline fee_calculator/apps/api/tests/test_advocate_type_api.py]: Mostly it's just that there should two blank lines between the final import statement and the first class or function definition. Like I say, super trivial.

File: fee_calculator/apps/calculator/tests/lib/utils.py [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+
```
Comment (ibrechin) [inline fee_calculator/apps/calculator/tests/lib/utils.py]: This blank line is unnecessary.

File: fee_calculator/apps/calculator/tests/lib/utils.py [hunk 3]
```diff
@@ -229,3 +232,24 @@ def scenario_id_to_clf(scenario_id):
...
+
+def prevent_request_warnings(original_function):
+    """
+    If we need to test for 404s or 405s this decorator can prevent the
+    request class from throwing warnings.
+    """
+    def new_function(*args, **kwargs):
+        # raise logging level to ERROR
+
```
Comment (ibrechin) [inline fee_calculator/apps/calculator/tests/lib/utils.py]: This blank line is also unnecessary.

File: fee_calculator/apps/api/tests/test_advocate_type_api.py (hunk 1 above)
Comment (jsugarman) [reply fee_calculator/apps/api/tests/test_advocate_type_api.py]: 👍 thanks for this

**Pull request merged**